smoke-test/tests/authenticated/teaspoons_pipeline_runs_list_tests.py

```diff
@@ -3,11 +3,11 @@
 
 class TeaspoonsPipelineRunsListTests(TeaspoonsSmokeTestCase):
     '''
-    Test the pipeline runs list endpoint for a 200 status code
+    Test the pipeline runs list v2 endpoint for a 200 status code
     '''
     @staticmethod
     def pipeline_runs_list_url() -> str:
-        return TeaspoonsSmokeTestCase.build_teaspoons_url("api/pipelineruns/v1/pipelineruns?limit=10")
+        return TeaspoonsSmokeTestCase.build_teaspoons_url("api/pipelineruns/v2/pipelineruns?limit=10")
```

Comment:
can we also change `limit` to `pageSize` since that was changed in the [v2 api](https://teaspoons.dsde-prod.broadinstitute.org/#/pipelineRuns/getAllPipelineRunsV2) as well